### DISCUSSION TITLE
Cake.ExtendedNuGet is outdated and does not support newer NuGet / .Net Framework

### DISCUSSION THREAD

**Discussion by MCCoolMoDAue** [cake-build/cake#3052] [Extension Q&A] [answered]
Hello,

Cake.ExtendedNuGet is using an outdated NuGet version.
We've noticed when we updated projects to net48. We've been using an older Cake version - 0.32.1
The NuGet version used in that old Cake version does nut support net48.
So we've updated to 0.38.1, but now the Cake.ExtendedNuGet does not work with the newer NuGet version from  the newer Cake.

I've tried contacting the maintainer, so far without success.
However, the documentation for that package is managed by the central cake webstite.

How do you ensure that the 3d party cake extensions to be up to date? is there some kind of process?

Can someone contact the maintainer, or is the package simply dead?

**Comment by nils-a** [comment, marked as answer]
There is no formal process to ensure that extension are always up to date, no.

Since this is all voluntary work in OSS sometimes maintainers are simply busy elsewhere, but I'd imagine pull requests are always welcome 😄

**Reply by MCCoolMoDAue** [reply]
Thx, i will give it a shot!

**Reply by augustoproiete** [reply]
PR from @MCCoolMoDAue fixes the NuGet dependency issue and was merged https://github.com/cake-contrib/Cake.ExtendedNuGet/pull/37

@nils-a is making [a number of improvements](https://github.com/cake-contrib/Cake.ExtendedNuGet/issues?q=author%3Anils-a+) to the addin to bring it up-to-speed with our best practices.

Version 0.3.0 to be released soon ( /cc @Redth )

**Reply by nils-a** [reply]
Version 3.0.0 has been released: https://www.nuget.org/packages/Cake.ExtendedNuGet/3.0.0
